### ISSUE TITLE
PUT not upsert in kong 2.0

### ISSUE THREAD

**Issue by luispabon** [phpdocker-io/kong-certbot-agent#48]
Kong docker image: kong:alpine [2.0.0; latest]
kong-certbot-agent image: kong-certbot-agent:latest [3.2.2]

Using the latest versions of the dockers, the agent can't put new certs on Kong. 
Return this error:

_replaced the domain with <domain_name>; it's just one registry_
```
Kong error: code 400, message Kong error 400: Client error: `PUT http://kong:8001/certificates/<domain_name>` resulted in a `400 Bad Request` response:
{"message":"schema violation (snis: <domain_name> is duplicated)","name":"schema violation","fields":{" (truncated...)
. Request method `PUT`, headers {"User-Agent":["GuzzleHttp\/6.5.1 curl\/7.58.0 PHP\/7.4.1"],"Content-Type":["application\/json"],"Host":["kong:8001"],"accept":["application\/json"]}, body "", domains <domain_name>
```

The validation passed without problems. And both containers are built thought docker-compose.

_Originally posted by @nicolasmontenegro in https://github.com/phpdocker-io/kong-certbot-agent/issues/14#issuecomment-582083433_

**Comment by zmeyc** [comment]
This became broken in Kong 1.5.0-alpine.
1.4.0-alpine works.

**Comment by luispabon** [comment]
Ah, that's annoying. That's a bc break right there without bumping the major version.

**Comment by luispabon** [comment]
@nicolasmontenegro how many certificates do you have on your kong installation? I'm trying to determine if that error above happens when you have two different certificates pointing to the same domain (sni).

**Comment by jschirrmacher** [comment]
I have the same problem with Kong 2.0.1, and in fact, I use a comma separated list of domain names. The first one is obviously used for the `PUT /certificates/<domain_name>` path variable of the call, but also included in the body which seems to be a problem with newer Kongs.

**Comment by luispabon** [comment]
Unfortunately kong 1.5 still affected, but I believe it is an issue on their end.

Opened a ticket here: https://github.com/Kong/kong/issues/5652

Testing 2.0 now

**Comment by okanozdemir** [comment]
Same error in Kong 1.4.3

**Comment by Igor-lkm** [comment]
So, any news with this issue?

Same error w/ kong `2.0.2`

**Comment by Igor-lkm** [comment]
Tested with kong version `2.0.3 `. Everything is ok, I was able to renew certificates.

**Comment by luispabon** [comment]
Apologies for the lack of feedback. As @Igor-lkm mentioned this is actually a kong (not a certbot-agent) issue and it's been fixed in kong 2.0.3.

No fix for the kong 1.x series is forthcoming I'm afraid.
